submission/proxy_test.go

```diff
@@ -176,8 +159,7 @@ Init:
 			if ok {
 				t.Fatalf("p.Refresh() after initial p.Run() invoked init-status signal, expected none")
 			}
-		case e := <-p.Errors:
-			t.Log(e)
+		case <-p.Errors:
```

Comment:
Why did this change back?

Reply:
Not convinced it's helpful for testing purposes.

Reply:
Why's that?

Reply:
How logging those would help testing the issue?

Reply:
If the test fails, the errors might give some clue as to why.